components/ui/form.tsx

```diff
@@ -78,7 +78,7 @@ const FormItem = React.forwardRef<
 
   return (
     <FormItemContext.Provider value={{ id }}>
-      <div ref={ref} className={cn("space-y-2", className)} {...props} />
+      <div ref={ref} className={cn("space-y-1.5", className)} {...props} />
```

Comment:
wont this effect all the forms? are they fixed or not?

Reply:
yes, in design it was 1.5, to i matched the design